src/work-allocation/components/task-manager-filter/task-manager-filter.component.ts

```diff
@@ -50,6 +50,10 @@ export class TaskManagerFilterComponent implements OnInit, OnDestroy {
           name: 'selectPerson',
           value: ['All']
         },
+        {
+          name: 'taskName',
+          value: ['']
```

Comment:
Why is this a Blank String?

Reply:
when it resets to default, tasknames defaults to nothing so that it brings the task list for all tasknames